packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -159,3 +159,19 @@ it('should hide the caret', () => {
   expect(element.getCssProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
 });
 ```
+
+## Dispatch Custom Event
+
+### Terra.dispatchCustomEvent
+Fire a custom event with `metadata`. Use this to manually update a deployed terra application's locale, or fire updates for a custom event listener. This method accepts an object containing the event `name` (string) and event `metaData` (object).
+
+```js
+it('should update an application's locale, () => {
+  const event = {
+    name: 'applicationBase.testOverride',
+    metaData: { locale: 'ar' }
+  };
+
+  Terra.dispatchCustomEvent(event);
```

Comment:
After reading this documentation it makes me wonder if there would be any benefit to exposing an explicit function that switches the locale. It would abstract away the  `dispatchCustomEvent`

```
Terra.setLocale('en');
```

Reply:
I like that idea, seeing how this utility was created for that purpose. we could expose a `setLocale` only? if someone requests `dispatchCustomEvent`, we could expose that later